src/DynamoPackagesUI/Utilities/PackageManagerCommands.cs

```diff
@@ -50,6 +51,46 @@ public bool UnloadPackage(Package localPackage)
 
             return true;
         }
-        
+
+        public MessageBoxResult Show(PackageManagerMessages msgID, string caption, MessageBoxButton options, MessageBoxImage boxImage, params object[] args)
```

Comment:
The method name should be ShowMessageBox, what are args? Please document all the public methods.